### ISSUE TITLE
_vendor/ missing from 0.0.9 on pypi

### ISSUE THREAD

**Issue by nthomas-mozilla** [di/pip-api#33]
We're hitting this error:
```
Traceback (most recent call last):
  File "/usr/local/bin/hashin", line 6, in <module>
    from hashin import main
  File "/usr/local/lib/python2.7/site-packages/hashin.py", line 20, in <module>
    import pip_api
  File "/usr/local/lib/python2.7/site-packages/pip_api/__init__.py", line 3, in <module>
    from pip_api._vendor.packaging import version as packaging_version
ImportError: No module named _vendor.packaging
```
The wheel and tar.gz on pypi are both missing `pip_api/_vendor/` which #31 added.

**Comment by di** [comment]
Sorry about that! [Just released 0.0.10](https://pypi.org/project/pip-api/0.0.10/).

**Comment by nthomas-mozilla** [comment]
Thanks, 0.10.0 is working for us.
